PMM-12116 Update dependencies with CVEs

0leksii opened the pull request:
Update dependencies with the CVEs found for exporters.

https://jira.percona.com/browse/PMM-12116

https://github.com/percona/postgres_exporter/pull/121
https://github.com/percona/node_exporter/pull/45
https://github.com/percona/mysqld_exporter/pull/126

0leksii committed:
PMM-12116 Update dependencies with CVEs

Update dependencies with the CVEs found for exporters.

JNKPercona commented:
server docker - perconalab/pmm-server-fb:PR-3266-27606c4
client docker - perconalab/pmm-client-fb:PR-3266-27606c4
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3266-27606c4.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3266-27606c4&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3266-27606c4.tar.gz

0leksii committed:
PMM-12116 Use mongodb_exporter main branch

Retrigger build with having main branch explicitly
set for the mongodb_exporter.

JNKPercona commented:
server docker - perconalab/pmm-server-fb:PR-3266-b52d703
client docker - perconalab/pmm-client-fb:PR-3266-b52d703
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3266-b52d703.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3266-b52d703&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3266-b52d703.tar.gz
